CHANGELOG.md

```diff
@@ -2,8 +2,23 @@
 
 ## Unreleased
 
+## 3.10.0
+
+### New Features
 * To keep up with the latest security protocols implemented by Amazon Web
-  Services, the agent now uses AWS IMDSv2 to find utilization data.
+  Services, the agent now uses [AWS
+  IMDSv2](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html)
+  to find utilization data. [#249](https://github.com/newrelic/go-agent/pull/249)
+
+### Changes
+* Updated the locations of our license files so that Go docs https://pkg.go.dev 
+  will display our agent. Thanks @tydavis for your PR to fix this! [#254](https://github.com/newrelic/go-agent/pull/254)
+* Added an Open Source repo linter GitHub action that runs on push. [#262](https://github.com/newrelic/go-agent/pull/2462)
```

Comment:
Link here should be to pull/262, not pull/2462.

Reply:
Thanks, fixed.